### ISSUE TITLE
Can Apiman support graphql

### ISSUE THREAD

**Issue by riginoommen** [apiman/apiman#1389]

**Comment by msavy** [comment]
Hi @riginoommen! We don't currently have explicit support for GraphQL. I've been thinking about it and how I might get something like that sponsored and what it might look like.

What are your thoughts on what Apiman should be able to do for something like GraphQL? i.e. what does API Management mean in that context.

**Comment by riginoommen** [comment]
Hi @msavy ,

Currently i see tht there is no tool which support GraphQL + REST APIs together. As an API Developer/Consumer it will be great to see a single UI where we can see the apis all together. Also there is no single UI for this aggregated view.

1) Single UI to manage the REST & GraphQL
2) Unified API Documention View

**Comment by msavy** [comment]
What would API Management of GraphQL look like. I'm not a GraphQL expert. Would the gateway need to treat it in a special way?

**Comment by riginoommen** [comment]
@msavy Yes exactly. Graphql uses a unified schema for the gateway approach. there are different methods of schema building in gateway 

* Schema stiching
* Federaton
